src/main/webapp/index.html

```diff
@@ -6,10 +6,21 @@
 </head>
 <body>
   <p></p>
+  <a></a>
   <script>
       fetch('/hello', {method: 'GET'}).then(response => response.text()).then((text) => {
           document.querySelector('p').innerText = text;
       });
+
+      fetch('/auth', {method: 'GET'}).then(response => response.json()).then((loggedInInfo) => {
+          let loginLogout = document.querySelector('a');
```

Comment:
it can be better to rename it to `authLinkElement`, because it says that this variable is and HTML Element (tag <a>) and this element provides authentication link (I found that authentication can be used as common word for login/logout)